/usr/ports/pobj/pdfpc-4.6.0/pdfpc-4.6.0/src/classes/drawings/drawing_commands.vala:57.46-57.59: error: `pdfpc.DrawingCommand' is not a supported generic type argument, use `?' to box value types
   57 |             this.drawing_commands = new List<DrawingCommand>();
      |                                              ^~~~~~~~~~~~~~    
/usr/ports/pobj/pdfpc-4.6.0/pdfpc-4.6.0/src/classes/drawings/drawing_commands.vala:58.43-58.56: error: `pdfpc.DrawingCommand' is not a supported generic type argument, use `?' to box value types
   58 |             this.redo_commands = new List<DrawingCommand>();
      |                                           ^~~~~~~~~~~~~~    
/usr/ports/pobj/pdfpc-4.6.0/pdfpc-4.6.0/src/classes/drawings/drawing_commands.vala:73.43-73.56: error: `pdfpc.DrawingCommand' is not a supported generic type argument, use `?' to box value types
   73 |             this.redo_commands = new List<DrawingCommand>(); // clear

Index: src/classes/drawings/drawing_commands.vala
--- src/classes/drawings/drawing_commands.vala.orig
+++ src/classes/drawings/drawing_commands.vala
@@ -54,8 +54,8 @@ namespace pdfpc {
         }
 
         public void clear() {
-            this.drawing_commands = new List<DrawingCommand>();
-            this.redo_commands = new List<DrawingCommand>();
+            this.drawing_commands = new List<DrawingCommand?>();
+            this.redo_commands = new List<DrawingCommand?>();
         }
 
         public void add_line(bool is_eraser,
@@ -70,7 +70,7 @@ namespace pdfpc {
 
             // After adding a new line you can no longer redo the old
             // path.
-            this.redo_commands = new List<DrawingCommand>(); // clear
+            this.redo_commands = new List<DrawingCommand?>(); // clear
 
             bool new_path = true;
             double epsilon = 1e-4; // Less than 0.1 pixel for a 1000x1000 img
